cog/slots.py

```diff
@@ -55,16 +58,39 @@ class Slots(customs.cog.Cog):
         }
 
 
+    _credits = {
+        'low' : 1,
+        'mid' : 5,
+        'high' : 15
+    }
+
+
+    # credits -> low = 1, mid = 5, high = 15
     @commands.group(alias=['slots'])
-    async def slots(self, ctx, credits):
+    async def slots(self, ctx, credits_index = 'low'):
```

Comment:
I suggest using a converter to check if credits_index is an acceptable parameter. Look into converters on the discord.py doc.